### PR TITLE
docs: Rename SoundEffect wave to waveform

### PR DESCRIPTION
**Pull request by microbit-carlos**
https://microbit-micropython--765.org.readthedocs.build/en/765/audio.html

### CODE REVIEW

File: docs/audio.rst [hunk 1]
```diff
@@ -129,8 +129,8 @@ Sound Effects **V2**
     :param duration: Duration of the sound (ms), default: ``500``
     :param vol_start: Start volume value, range 0-255, default: ``255``
     :param vol_end: End volume value, range 0-255, default: ``0``
-    :param wave: Type of wave shape, one of these values: ``WAVE_SINE``,
-        ``WAVE_SAWTOOTH``, ``WAVE_TRIANGLE``, ``WAVE_SQUARE``,
+    :param waveform: Type of waveform shape, one of these values:
```
Comment (microbit-carlos) [inline docs/audio.rst:132]: A suggestion by @microbit-giles

Comment (dpgeorge) [reply docs/audio.rst:132]: I suggest we then also change the constants to `WAVEFORM_SAWTOOTH` etc.

**Comment by microbit-carlos** [comment]
@dpgeorge I've separated the predefined sounds out of this PR into a new one, as we won't have time to implement these before the v2.1.0 release:
- https://github.com/bbcmicrobit/micropython/pull/768

This PR now only contains the `wave`->`waveform` rename, and it's ready to be implemented.

File: docs/audio.rst (hunk 1 above)
Comment (dpgeorge) [reply docs/audio.rst:132]: I think consistency is important here.

**Comment by dpgeorge** [comment]
This change has now been made in the v2 code.

**Comment by microbit-carlos** [comment]
I'll merge this, as it has been released with the v2.1.0-beta.3.

**Pull request merged**